website/src/content/deployment-examples/azure.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Deploying to Microsoft Azure
+---
+
+This deployment example refers to [Microsoft Azure](https://azure.microsoft.com/).
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [Microsoft Azure](https://azure.microsoft.com/free/) account.
+- A [Github](https://github.com/join) account.
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, make sure to provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) that you got when you configured the Custom Application in the Merchant Center.
+
+Moreover, you need to provide the [production URL](/api-reference/application-config#envproductionurl) from your Microsoft Azure project. You can keep the standard Azure URL `https://<project>.1.azurestaticapps.net` or provide your own custom domain.
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<project>.1.azurestaticapps.net"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<project>.1.azurestaticapps.net"
+    }
+  }
+}
+```
+
+Create the environment variables in the Microsoft Azure application settings: **Home > Static Web Apps > Your project name > Configuration > Application Settings**.
```

Comment:
Here we are mentioning the `Static Web Apps` before mentioning (in the next section) that this is our preferred way of integration in Azure.
Not sure if it we would need to move this part to the next section.

Reply:
Good point. This is actually the same structure as in some of the other examples.

If we want to change it, please remember to also adjust the other related documents.

Reply:
Sure, this makes sense. I updated the structure. Please let me know if you still consider that out of place after the changes